### PR TITLE
Services ECS, Schema Registry, and Schema Registry UI under master template

### PR DESCRIPTION
**Pull request by dloiko**
#### Description

This change moves the Services template under the master template and creates the schema registry service and wires it up to all the moving parts. As well, I am hardcoding the ASG params for ZK and kafka as we are only running 3 brokers and 3 zk nodes. 

There will be a partner PR for the infra-config repo that I will put in as well to add the service params.

### CODE REVIEW

File: deploy_master_stack [hunk 1]
```diff
@@ -12,12 +12,10 @@ fi
 
 readonly CFNS=$(grep \
     --include="*.yaml" \
-    --exclude=services-template.yaml \
     --exclude=master.yaml \
     --exclude=services.yaml \
     --exclude=efs.yaml \
```
Comment (hackmad) [inline deploy_master_stack]: `services.yaml` can be removed since it no longer exists

File: ecs.yaml [hunk 2]
```diff
@@ -40,6 +44,10 @@ Parameters:
     AllowedValues: ['true', 'false']
     Default: true
     ConstraintDescription: Must be either 'true' or 'false'
+  AdditionalSecurityGroup:
+    Type: AWS::EC2::SecurityGroup::Id
+    Description: Additional security group to add ECS instances to
+    Default: ''
```
Comment (hackmad) [inline ecs.yaml]: Strange you were able use `AWS::EC2::SecurityGroup::Id` with a default value of empty string. Did this not work for `ELBSecurityGroupId`?

File: services-template.yaml [hunk 3]
```diff
@@ -23,9 +23,6 @@ Parameters:
   VPCId:
```
Comment (hackmad) [inline services-template.yaml]: Do we still need this template?

Comment (dloiko) [reply services-template.yaml]: Yes, the master template references the services-template in order to deploy all the components for the services.

File: deploy_master_stack (hunk 1 above)
Comment (dloiko) [reply deploy_master_stack]: Will change

File: ecs.yaml (hunk 2 above)
Comment (dloiko) [reply ecs.yaml]: It should work for `ELBSecurityGroupId` as well - will test and change it to match this one.

**Pull request merged**